pkgs/servers/http/nginx/modules.nix

```diff
@@ -67,19 +67,14 @@ in
   };
 
   brotli = {
-    src = let gitsrc = pkgs.fetchFromGitHub {
+    src = fetchFromGitHub {
       name = "brotli";
       owner = "google";
       repo = "ngx_brotli";
-      rev = "25f86f0bac1101b6512135eac5f93c49c63609e3";
-      sha256 = "02hfvfa6milj40qc2ikpb9f95sxqvxk4hly3x74kqhysbdi06hhv";
-    }; in
-      pkgs.runCommand "ngx_brotli-src" { } ''
-        cp -a ${gitsrc} $out
-        substituteInPlace $out/filter/config \
-          --replace '$ngx_addon_dir/deps/brotli/c' ${lib.getDev pkgs.brotli}
-      '';
-    inputs = [ pkgs.brotli ];
+      fetchSubmodules = true;
```

Comment:
```suggestion
```
we want to use our brotli if possible to not rely on commits like https://github.com/google/ngx_brotli/commit/6e975bcb015f62e1f303054897783355e2a877dc